docsrc/sample.rst

```diff
@@ -32,7 +33,7 @@ NUTS-HMC sampler configuration
 
 - ``save_warmup``: When ``True``, sampler saves warmup draws as part of output csv file.
 
-- ``thin``: Period between saved samples.
+- ``thin``: Period between saved samples (draws).  Default is 1, i.e., save all iterations.
```

Comment:
Should this be readable at the output? `fit.thin` for instance

Reply:
it's available in that it's in the dict returned by propery `cmdstan_config`.  trying to avoid interface code clutter.  OTOH, the more that's in the docs, the better, so will add more to docs.

Reply:
agreed - added this as a property.